website/docs/jellyfish/api/blockchain.md

```diff
@@ -257,3 +257,60 @@ interface MempoolTx {
   }
 }
 ```
+
+## getBlockStats
+
+Get block statistics for a given window. Returns all stats values if nothing is passed in the second param.  
+
+
+```ts title="client.blockchain.getBlockStats()"
+
```

Comment:
![image](https://user-images.githubusercontent.com/4266087/119652286-74bee880-be58-11eb-838d-f709d3cd155e.png)

Because of your line break above and below the code docs have them. You need to be aware of this.